### ISSUE TITLE
Visualisation The page is empty

### ISSUE THREAD

**Issue by autumn-2-net** [spfrommer/torchexplorer#38]
![图片](https://github.com/spfrommer/torchexplorer/assets/109412646/c7a13179-9a48-4d73-b2f5-5f885c9feea5)
li'ke this

[vega_dataless.json](https://github.com/spfrommer/torchexplorer/files/13748376/vega_dataless.json)
this is vega_dataless.json

**Comment by spfrommer** [comment]
Would you mind sharing the python source to reproduce? By itself, vega_dataless won't show anything besides the "help" question mark because there's no data.

Note that depending on your hardware it may take a few seconds for the explorer to "load". Working on making the interface give better feedback for this: https://github.com/spfrommer/torchexplorer/issues/39

**Comment by autumn-2-net** [comment]
> Would you mind sharing the python source to reproduce? By itself, vega_dataless won't show anything besides the "help" question mark because there's no data.
> 
> Note that depending on your hardware it may take a few seconds for the explorer to "load". Working on making the interface give better feedback for this: #39

However I've been waiting on the page for over 5 minutes and have also tried other browser kernels

my code https://github.com/autumn-2-net/code

**Comment by spfrommer** [comment]
Thanks for sharing--what's the source for:

`from moduls.codebook import CodeBook, VQCodebook`

in `hifivae.py`?

Also, does the following work for you:

```python
import torch
import torchvision
import torchexplorer

model = torchvision.models.resnet18(pretrained=False)
dummy_X = torch.randn(5, 3, 32, 32)

# Only log input/output and parameter histograms, if you don't want even these set log=[].
torchexplorer.watch(model, log_freq=1, log=['io', 'params'], backend='standalone')
# Do one forwards and backwards pass
model(dummy_X).sum().backward()
# Your model will be available at http://localhost:5000
```

**Comment by autumn-2-net** [comment]
> Thanks for sharing--what's the source for:
> 
> `from moduls.codebook import CodeBook, VQCodebook`
> 
> in `hifivae.py`?
> 
> Also, does the following work for you:
> 
> ```python
> import torch
> import torchvision
> import torchexplorer
> 
> model = torchvision.models.resnet18(pretrained=False)
> dummy_X = torch.randn(5, 3, 32, 32)
> 
> # Only log input/output and parameter histograms, if you don't want even these set log=[].
> torchexplorer.watch(model, log_freq=1, log=['io', 'params'], backend='standalone')
> # Do one forwards and backwards pass
> model(dummy_X).sum().backward()
> # Your model will be available at http://localhost:5000
> ```

i forget to upload it  https://github.com/autumn-2-net/code/blob/main/codebook.py

**Comment by autumn-2-net** [comment]
> ```python
> import torch
> import torchvision
> import torchexplorer
> 
> model = torchvision.models.resnet18(pretrained=False)
> dummy_X = torch.randn(5, 3, 32, 32)
> 
> # Only log input/output and parameter histograms, if you don't want even these set log=[].
> torchexplorer.watch(model, log_freq=1, log=['io', 'params'], backend='standalone')
> # Do one forwards and backwards pass
> model(dummy_X).sum().backward()
> # Your model will be available at http://localhost:5000
> ```

![图片](https://github.com/spfrommer/torchexplorer/assets/109412646/5cafeb79-8aed-41b6-8111-2554469bb89e)
it work

**Comment by spfrommer** [comment]
Found the floating point bug, thanks for the report! It runs now on my end. Released the bugfix as 1.1.1, try upgrading and let me know if it doesn't work.
